src/storageMaps/details/utils.ts

```diff
@@ -51,7 +50,7 @@ export const transformFormValuesToK8sSpec = (
           offloadPlugin: {
             vsphereXcopyConfig: {
               secretRef: mapping.storageSecret,
-              storageVendorProduct: mapping.storageProduct,
+              storageVendorProduct: mapping.storageProduct as 'vantara' | 'ontap' | 'primera3par',
```

Comment:
please create a type for this, and don't we have more vendors?

Reply:
We have more vendors, but the current type we use from `@kubev2v/types` only specifies these 3, which is why we're casting here. So the type already exists, its just not current to what vendor products we offer.